### DISCUSSION TITLE
Toggling wordwrap discards collapsed state

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#45070] [Mobile] [answered]
### Select Topic Area

Bug

### Body

GitHub Mobile beta

1. Open https://github.com/actions/cache/pull/1035
2. Tap `32 files changed`
3. Collapse `.devcontainer/devcontainer.json`... `package.json`
4. Tap the ⚙️ 
5. Toggle `wrap lines`
6. Tap 🔙

### Expected results
Items collapsed in 3 are still collapsed at 6

### Actual results
The items collapsed in 3 are reverted by 6

**Comment by iskae** [comment, marked as answer]
Hi @jsoref 👋 

Yep, I can also reproduce this behaviour in our Android App 👍 Just created an issue for it and we will be fixing it asap 🙇 

Thanks for reporting this bug ❤️ Please stay tuned for our future versions for the fix 😊
